Tutorials/Multilanguage-Setup/index.md

```diff
@@ -4,10 +4,10 @@
 _Note: This will be shown on Umbraco 7.8.1, newer and older versions may differ. Also this assumes you already have a site with content in one language ready to work with._
 
 ### First step 
-When setting up a multilingual site is to copy the content you have in the original language. To do this, chose the node you want to copy - in this example everything is copied at once by copying the Home node, left click Home and click copy in the menu.
```

Comment:
Shouldn't it still be *When* rather than *when* as it is a new paragraph after the headline?

Reply:
I'd prefer that, but the author did it further down too on another step. Like the paragraph is leading on from the title. I can change it if you like on this one and the other paragraph.

Reply:
Good fix for the *Second step* section ;)